docassemble/ALWeaver/data/templates/output.mako

```diff
@@ -0,0 +1,370 @@
+<%doc>
+    Initial metadata and includes
+</%doc>
+---
+include:
+  - docassemble.AssemblyLine:assembly_line.yml
+  % for include in get_yml_deps_from_choices( interview.jurisdiction_choices.true_values() + interview.org_choices.true_values()):
+  - ${ include }
+  % endfor
+---
+metadata:
+  title: |
+    ${ interview.title }
+  short title: |
+    ${ interview.short_title }
+  % if interview.categories.any_true():
+  tags:
+    % for category in interview.categories.true_values():
+    - ${ category }
+    % endfor
+  % endif
+  authors:
+    % for author in interview.author.splitlines():
+    - ${ author }
+    % endfor
+---
+mandatory: True
+comment: |
+  Global interview metadata
+variable name: interview_metadata["${ interview_label }"]
+data:
+  al_weaver_version: "${ package_version_number }"
+  generated on: "${ today().format("yyyy-MM-dd") }"
+  title: >-
+    ${ oneline(interview.title) }
+  short title: >-
+    ${ oneline(interview.short_title) }
+  description: |-
+${ indent(interview.description, by=4) }
+  original_form: >-
+${ indent(interview.original_form, by=4) }
+  % if len(interview.allowed_courts.true_values()) < 1:
+  allowed courts: []
+  % else:
+  allowed courts: 
+    % for court in interview.allowed_courts.true_values():
+    - "${ escape_double_quoted_yaml(oneline(court)) }"
+    % endfor
+  % endif
+  % if len(interview.categories.true_values()) < 1:
+  categories: []
+  % else:
+  categories:
+    % for category in set(interview.categories.true_values()) - {'Other'}:
+    - "${ escape_double_quoted_yaml(oneline(category)) }"
+    % endfor
+    % if interview.categories['Other']:
+    % for category in interview.other_categories.split(','):
+    - "${ escape_double_quoted_yaml(oneline(category.strip())) }"
+    % endfor
+    % endif
+  % endif
+  % if interview.typical_role:
+  typical role: "${ escape_double_quoted_yaml(oneline(interview.typical_role)) }"
```

Comment:
When doing a no template interview, this changed from `typical role: anonymous` to asking the user what the role is (I answered plaintiff). So it seems that the [block that sets "no template" defaults](https://github.com/SuffolkLITLab/docassemble-ALWeaver/blob/8df791433ee25c8b3d13f494c6e1bc41d455d7e1/docassemble/ALWeaver/data/questions/assembly_line.yml#L2188-L2196) isn't being executed for some reason.

Reply:
I think that the `typical role: anonymous` was actually a mistake. the prompt for user role is on the page that also defines `people_variables`. And I don't think there's a problem asking for this value. The fact that there's no output file doesn't really mean that whether the user is initiating an action or not is relevant.